webapp/src/pages/EditAvatarPage/EditAvatarPage.css

```diff
@@ -14,9 +14,25 @@
   overflow: scroll;
 }
 
+.edit-avatar-page .editable-avatar {
+  align-self: flex-start;
```

Comment:
for mobile looks much better as a grid, than as a flex with space between, so I'll not change it. I'm following figma.